renku/core/models/datasets.py

```diff
@@ -898,23 +899,26 @@ def generate_default_name(dataset_title, dataset_version=None):
     if is_dataset_name_valid(dataset_title):
         return dataset_title
 
-    name = re.sub(r"\s+", " ", dataset_title)
-    name = name.lower()[:24]
-
-    def to_unix(el):
-        """Parse string to unix friendly name."""
-        parsed_ = re.sub("[^a-zA-Z0-9]", "", re.sub(r"\s+", " ", el))
-        parsed_ = re.sub(" .+", ".", parsed_.lower())
-        return parsed_
-
-    name = [to_unix(el) for el in name.split()]
-    name = [el for el in name if el]
+    slug = get_slug(dataset_title)
+    slug = slug.lower()[:24]
 
     if dataset_version:
-        version = to_unix(dataset_version)
-        return "{0}_{1}".format("_".join(name), version)
-
-    return "_".join(name)
+        version_slug = get_slug(dataset_version)
+        return f"{slug}_{version_slug}"
```

Comment:
Should we change this to use something like `slug[:-len(version_slug)]` and limit version_slug length to something like 10 to ensure a consistent slug max length?